authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/broker/dao/pages/catalog/CatalogQueryDataOfferFetcher.kt

```diff
@@ -53,6 +53,7 @@ class CatalogQueryDataOfferFetcher(
         val query = DSL.select(
             d.ASSET_ID.`as`("assetId"),
             d.ASSET_TITLE.`as`("assetTitle"),
+            d.UI_ASSET_JSON.`as`("assetUiJson"),
```

Comment:
There is no need to fetch the whole assetJson. You can just use `fields.dataSourceAvailability` which extracts the value from the JSON in the DB itself saving some resources. Check out the class `CatalogQueryFields` for more info